### PR TITLE
chore: fix CI errors on javadoc

### PR DESCRIPTION
**Pull request by BigTailWolf**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/google-auth-library-java/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

If you write sample code, please follow the [samples format](
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md).

### CODE REVIEW

File: oauth2_http/java/com/google/auth/oauth2/AwsCredentials.java [hunk 1]
```diff
@@ -268,7 +268,8 @@ public Builder setAwsSecurityCredentialsSupplier(
      * Sets the AWS regional credential verification URL. If set, will override any credential
      * verification URL provided in the credential source. If not set, the credential verification
      * URL will default to
-     * https://sts.{region}.amazonaws.com?Action=GetCallerIdentity&Version=2011-06-15"
+     *
+     * <p>{@code https://sts.{region}.amazonaws.com?Action=GetCallerIdentity&Version=2011-06-15}
```
Comment (lsirac) [inline oauth2_http/java/com/google/auth/oauth2/AwsCredentials.java:272]: Use <a href=

File: oauth2_http/java/com/google/auth/oauth2/AwsCredentials.java [hunk 2]
```diff
@@ -268,7 +268,8 @@ public Builder setAwsSecurityCredentialsSupplier(
      * Sets the AWS regional credential verification URL. If set, will override any credential
      * verification URL provided in the credential source. If not set, the credential verification
      * URL will default to
-     * https://sts.{region}.amazonaws.com?Action=GetCallerIdentity&Version=2011-06-15"
+     *
+     * <p><a href=https://sts.{region}.amazonaws.com?Action=GetCallerIdentity&Version=2011-06-15>
```
Comment (clundin25) [inline oauth2_http/java/com/google/auth/oauth2/AwsCredentials.java:272]: Should <p> and <a href> have a closing tag as well?

**Review by clundin25 — APPROVED** [review]

**Review by lsirac — APPROVED** [review]

File: oauth2_http/java/com/google/auth/oauth2/AwsCredentials.java (hunk 2 above)
Comment (BigTailWolf) [reply oauth2_http/java/com/google/auth/oauth2/AwsCredentials.java]: Was not intentionally using <p>, let me try to remove it.

Comment (BigTailWolf) [reply oauth2_http/java/com/google/auth/oauth2/AwsCredentials.java]: Hmmmm. lint is doesn't allow me to remove it and failed. Adding a closing tag also failed it.

**Pull request merged**